integration_tests/suite/base/test_call_pickup_target_group.py

```diff
@@ -106,6 +106,34 @@ def test_associate_multi_tenant(
     response.assert_match(400, e.different_tenant())
 
 
+@fixtures.call_pickup()
+@fixtures.user()
+@fixtures.group()
+@fixtures.group()
+def test_interceptor_user_has_targets(call_pickup, user1, group1, group2):
```

Comment:
don't add number suffix if there are only one resource
```suggestion
def test_interceptor_user_has_targets(call_pickup, user, group1, group2):
```